template_feed/Microsoft.DotNet.Common.ProjectTemplates.1.x/content/ClassLibrary-CSharp/.template.config/localize/templatestrings.fr.json

```diff
@@ -1,21 +1,21 @@
 ﻿{
   "author": "Microsoft",
-  "name": "Class Library",
+  "name": "Bibliothèque de classe",
   "description": "Projet de création d'une bibliothèque de classes ciblant .NET Standard ou .NET Core",
-  "symbols/TargetFrameworkOverride/description": "Overrides the target framework",
-  "symbols/Framework/description": "The target framework for the project.",
-  "symbols/Framework/choices/netcoreapp1.0/description": "Target netcoreapp1.0",
-  "symbols/Framework/choices/netcoreapp1.1/description": "Target netcoreapp1.1",
-  "symbols/Framework/choices/netstandard1.0/description": "Target netstandard1.0",
-  "symbols/Framework/choices/netstandard1.1/description": "Target netstandard1.1",
-  "symbols/Framework/choices/netstandard1.2/description": "Target netstandard1.2",
-  "symbols/Framework/choices/netstandard1.3/description": "Target netstandard1.3",
-  "symbols/Framework/choices/netstandard1.4/description": "Target netstandard1.4",
-  "symbols/Framework/choices/netstandard1.5/description": "Target netstandard1.5",
-  "symbols/Framework/choices/netstandard1.6/description": "Target netstandard1.6",
-  "symbols/langVersion/description": "Sets langVersion in the created project file",
-  "symbols/skipRestore/description": "If specified, skips the automatic restore of the project on create.",
-  "postActions/restore/description": "Restore NuGet packages required by this project.",
-  "postActions/restore/manualInstructions/default/text": "Run 'dotnet restore'",
-  "postActions/open-file/description": "Opens Class1.cs in the editor"
+  "symbols/TargetFrameworkOverride/description": "Remplace l’infrastructure cible",
+  "symbols/Framework/description": "Framework cible du projet.",
+  "symbols/Framework/choices/netcoreapp1.0/description": "Netcoreapp1.0 cible",
```

Comment:
@cristianosuzuki77 this is an issue: translators should not translated TFM part: 'netcoreapp1.0', 'netstandard2.0' etc. That applies to all similar files below.

@jonfortescue @bekir-ozturk is there a way to put a comment visible to translators like in resources?

Reply:
I know how to do it in XLF files but not in JSON files, unfortunately. Make sure you file an issue with the loc folks for this so they can fix the translations/give feedback to the translators

Reply:
I've created https://ceapex.visualstudio.com/CEINTL/_workitems/edit/448431 for this.
What would be expected action for this PR as per localization workflow? Should we merge it despite of the issue? There are 2 new PRs coming: https://github.com/dotnet/templating/pull/3349 and https://github.com/dotnet/templating/pull/3350, with same issue.

Reply:
Yeah, OLB creates a lot of noise that way because it creates a new PR on each run rather than updating an existing PR. Personally, I'd merge in spite of the issue and expect a fix to come further down the line.

Reply:
@jonfortescue thanks, it makes sense. I'll merge latest PR.

Reply:
A bit late, but I have just seen this. The proper way to add comments is to modify the key in the following way:
`"_symbols/Framework/choices/netcoreapp1.0/description.comment": "Do not translate netcoreapp1.0"`

Basically:
- We prepend the key with "_"
- We append ".comment"